console/src/components/WelcomePopup/WelcomePopup.less

```diff
@@ -1,15 +1,19 @@
 // WelcomePopup.less
 //
+@guideColor: rgba(255, 255, 255, 0.5);
```

Comment:
Please change this to `@guide-border-color` to match up with our LESS variable naming convention.